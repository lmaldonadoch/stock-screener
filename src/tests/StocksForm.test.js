import React from 'react';
import renderer from 'react-test-renderer';
import Form from '../containers/StocksForm';
import Wrapper from './Wrapper';

test('Form i rendered', () => {
  const component = renderer.create(
    <Wrapper>
      <Form />
    </Wrapper>
  );

  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
