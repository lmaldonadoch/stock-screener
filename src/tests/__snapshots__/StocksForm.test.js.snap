// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form i rendered 1`] = `
<form
  className="books-form"
  onSubmit={[Function]}
>
  <input
    className="title-input"
    name="title"
    onChange={[Function]}
    placeholder="Search Symbol"
    type="text"
  />
  <button
    type="submit"
  >
    <i
      className="fas fa-chart-line"
    />
  </button>
</form>
`;
